cpp/log/cert.cc

```diff
@@ -92,6 +93,24 @@ Cert::Status Cert::LoadFromDerString(const std::string& der_string) {
   return Cert::TRUE;
 }
 
+Cert::Status Cert::LoadFromDerBio(BIO *bio_in) {
+  if (x509_ != NULL) {
```

Comment:
at least new code should prefer `nullptr`, if you'd like to convert the others too then bonus points will be awarded! (but maybe do that in a followup PR if possible)